js/src/immersive-reader-sdk.ts

```diff
@@ -5,7 +5,17 @@ import { renderButtons } from './renderButtons';
 import { close, launchAsync } from './launchAsync';
 
 window.addEventListener('load', () => {
+    if (!('Promise' in window)) {
```

Comment:
Would it be more efficient to test against `!window.hasOwnProperty('Promise')`

Reply:
That's interesting -- I didn't know there was a difference, but apparently `in` looks at the inherited properties as well. You're right, hasOwnProperty should be a little more efficient. I tested with hasOwnProperty and everything worked fine in IE 11 so I'll make the change.